### DISCUSSION TITLE
Label of Level Not Added on PR | GSSOC Ext | #251

### DISCUSSION THREAD

**Discussion by Shariq2003** [codeaashu/DevDisplay#258] [General]
@codeaashu please look into this, my PR got merge without any label on it, please add labels.

PR : #251

**Comment by codeaashu** [comment]
> @codeaashu please look into this, my PR got merge without any label on it, please add labels.
> 
> PR : #251

Hi @Shariq2003, thanks for your interest in contributing to DevDisplay!

Your PR will be merged now, but we noticed you adding your profile on #DevDisplay but forgot to star the repo. It's like making an amazing sandwich and forgetting to take a bite! -- I think our repo's feeling a little lonely without your 🤗 So, be sure to star the DevDisplay GitHub Repository. 🌟

💌 This is a mandatory step for all contributors. & It's is the first priority of contribution in DevDisplay 💌

Thanks 👍🏻
